mlos_bench/mlos_bench/environments/composite_env.py

```diff
@@ -207,7 +205,7 @@ def run(self) -> Tuple[Status, datetime, Optional[Dict[str, TunableValue]]]:
 
         Returns
         -------
-        (status, timestamp, output) : (Status, datetime, dict)
+        (status, timestamp, output) : (Status, datetime.datetime, dict)
```

Comment:
same as above. since we import `from datetime import datetime` I would expect 
```suggestion
        (status, timestamp, output) : (Status, datetime, dict)
```
here

Reply:
I would have expected that too, but unfortunately the docstring parser is not perfect wrt to its static analysis, so I "solved" it by just being explicit whenever it complained about missing references.